openapi/components.yml

```diff
@@ -78,7 +78,6 @@ components:
           properties:
             application:
               type: object
-              additionalProperties: true
```

Comment:
Do we definitely want to remove this? Might leaving this make us more defensive to changes in future?

Reply:
We tested by adding additional keys to meta data & application level. The validation doesn't care about additional keys